test/e2e/tokentimeout_test.go

```diff
@@ -95,7 +95,7 @@ func TestTokenInactivityTimeout(t *testing.T) {
 		testTokenValidity(t, req, http.StatusOK, tokenWithoutTimeout.Name, "token with out timeout should work")
 	}
 
-	configInactivityTimeout := int32(600)
+	configInactivityTimeout := int32(420)
```

Comment:
why still that long? Why not `O(5 minutes)` ?

Reply:
Or iow: what's the reason for these two numbers?

Reply:
300s is the minimal, I think the original test was trying to prove that one of the timeouts apply and it's not the other one, so it had different values. I thought 2 minutes should be a sufficient-enough difference.